lib/pacemaker/pcmk_sched_instances.c

```diff
@@ -1493,8 +1497,8 @@ can_interleave_actions(const pcmk_action_t *first, const pcmk_action_t *then)
         return false;
     }
 
-    if (pcmk__ends_with(then->uuid, "_stop_0")
-        || pcmk__ends_with(then->uuid, "_demote_0")) {
+    if (g_str_has_suffix(then->uuid, "_stop_0")
```

Comment:
Should use CPP constants here too while we're modifying these lines, but I'm not gonna go back and bother with it unless you request other changes on this PR.